ddl/ddl.go

```diff
@@ -1542,19 +1548,16 @@ func processJobs(process func(*sess.Session, *model.Job, model.AdminCommandOpera
 			err = process(ns, job, byWho)
 			if err != nil {
 				errs[i] = err
-				break
+				continue
 			}
 
 			err = updateDDLJob2Table(ns, job, true)
 			if err != nil {
-				break
+				// unexpected error on updating the job table, should roll it back
```

Comment:
Maybe we encounter write-conflict error(If the job is running which is likely to happen). I think it also can do it like line1550-1551.

Reply:
Such kind of failures look like more serious. And the behavior  of writing to TiKV should keep the same as line 1533-1536. Also, we could handle the two returned value in different ways.

Reply:
If all (many) DDL operations are suspended, there is a risk of write-conflict error, and it is possible to try ten times and fail ten times. 
Originally, there was no exit, and it would have been easier for an interface to handle error in the same way as error. 
Or we could do something special here.